adapters/InMobi/InMobiAdapter/GADMAdapterInMobiInterstitialAd.m

```diff
@@ -102,7 +102,12 @@ - (void)requestInterstitialAd {
   if (extras && extras.keywords) {
     [_interstitialAd setKeywords:extras.keywords];
   }
-
+  
+  if (_interstitialAdConfig.watermark != nil) {
+    IMWatermark *watermark = [[IMWatermark alloc] initWithImageData:_interstitialAdConfig.watermark];
+    [_interstitialAd setWatermarkWith:watermark];
+  }
```

Comment:
Nit: Our internal code review system detects empty spaces in this line, can you remove them?